qa/aws-ec2-mgr/ReadMe.md

```diff
@@ -0,0 +1,86 @@
+AWS EC2 Instance Manager - Deploy/Terminate Helper Script
```

Comment:
Rename file to `README.md` to stick with convention.

Reply:
Done.